pkg/controller/kieapp/constants/constants.go

```diff
@@ -35,8 +35,8 @@ var VersionConstants = map[string]*api.VersionConfigs{
 		DatagridComponent:   Datagrid8Component,
 		MySQLImageURL:       MySQL80ImageURL,
 		MySQLComponent:      MySQL80Component,
-		PostgreSQLImageURL:  PostgreSQL12ImageURL,
-		PostgreSQLComponent: PostgreSQL12Component,
```

Comment:
shouldn't we move this to the prior version or use PSSQL 13 in the prior version as well?

Reply:
ok